contrib/crane/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "crane"
+pkgver = "0.20.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
```

Comment:
`make_build_args = ["./cmd/crane"]`

Reply:
ok, but note `black` does not care and the current version is better for diffing changes

Reply:
it looks worse, and the "diffing changes" is so inconsequential and unlikely to ever matter in any way that it might as well not be a thing

Reply:
agree on "looks worse" and "is so inconsequential"; not having a strong opinion here